### ISSUE TITLE
PSA: Waking from Sleep() via External Interrupt

### ISSUE THREAD

**Issue by DirtyEngineer** [MCUdude/MegaCore#97]
TL;DR - Only use INT3:0 to wake from Sleep().

After several nights of beating my head against the wall trying to get the Atmega128 to wake from sleep using INT6 I finally RTFM.

![power-down mode](https://user-images.githubusercontent.com/10263751/47938211-56b3c600-deb1-11e8-851f-e16aae8b4d59.PNG)

![external interrupts hl](https://user-images.githubusercontent.com/10263751/47938221-5ca9a700-deb1-11e8-8b70-7301ff50f1ce.PNG)

Blarg - Reading the documentation seems to suggest that you can only wake from sleep using INT3:0. 

To test this I used the sketch below for INT7:0 with the LOW, FALLING, HIGH, RISING, CHANGE parameters.

[ATMega128_Interupt_Sleep_Test.zip](https://github.com/MCUdude/MegaCore/files/2544024/ATMega128_Interupt_Sleep_Test.zip)

Turns out you can only reliably wake from sleep using INT3:0 with the LOW or FALLING interrupt parameters.

I was able to wake sporadically using INT3:0 with the HIGH and RISING parameter but it takes quite some time (up to several seconds) and was not at all reliable.

The data sheet suggests that you can wake from Sleep() using INT7:4 but I was unsuccessful on all attempts using all parameters. 

Might I suggest a slight change in the documentation so this issue can be avoided?

Also including an example sketch in the documentation for putting the MC to sleep and waking up would be really helpful.  I wasn't able to find many examples online using the additional WIRING functions. Feel free to modify the included sketch for use as an example. I'm unsure if this is the correct way to sleep the Atmega128 but it seemed to work for me.

**Comment by MCUdude** [comment]
Thanks for pointing this out!
